In [1]:
pip install ultralytics

Note: you may need to restart the kernel to use updated packages.


In [2]:
from ultralytics import YOLO

models = [
    "yolo11n-seg.pt",
    "yolo11s-seg.pt",
    "yolo11m-seg.pt",
    "yolo11l-seg.pt",
    "yolo11x-seg.pt",
    "yolov9e-seg.pt"
]

data_yaml = "dataset_path.yaml"

for model_path in models:
    print(f"\nEvaluating {model_path}")
    model = YOLO(model_path)

    results = model.val(data=data_yaml, split="val", save=False)

    map50 = results.seg.map50
    print(f"mAP@0.5 (Seg) for {model_path}: {map50:.4f}")




Evaluating yolo11n-seg.pt
Ultralytics 8.3.146  Python-3.10.16 torch-2.7.0+cpu CPU (AMD Ryzen 5 5625U with Radeon Graphics)
YOLO11n-seg summary (fused): 113 layers, 2,868,664 parameters, 0 gradients, 10.4 GFLOPs
val: Fast image access  (ping: 0.00.0 ms, read: 1616.2587.9 MB/s, size: 286.6 KB)


val: Scanning C:\Users\tiess\PycharmProjects\Tygron\ModelResearch\Yolo\OnderzoekDataSubsetYolo\Val\labels.cache... 150 images, 9 backgrounds, 0 corrupt: 100%|██████████| 150/150 [00:00<?, ?it/s]
C:\Users\tiess\anaconda3\envs\machlearn\lib\site-packages\torch\utils\data\dataloader.py:665: UserWarning: 'pin_memory' argument is set as true but no accelerator is found, then device pinned memory won't be used.
  warnings.warn(warn_msg)
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):   0%|          | 0/10 [00:00<?, ?it/s]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validati

                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  10%|█         | 1/10 [00:02<00:24,  2.75s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validati

                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  20%|██        | 2/10 [00:06<00:25,  3.23s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validati

                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95): 100%|██████████| 10/10 [00:28<00:00,  2.86s/it]


                   all        150       1067     0.0177     0.0843       0.01    0.00379     0.0131     0.0628     0.0074    0.00219
                person        141       1067     0.0177     0.0843       0.01    0.00379     0.0131     0.0628     0.0074    0.00219
Speed: 1.9ms preprocess, 116.3ms inference, 0.0ms loss, 18.6ms postprocess per image
Results saved to runs\segment\val35
mAP@0.5 (Seg) for yolo11n-seg.pt: 0.0074

Evaluating yolo11s-seg.pt
Ultralytics 8.3.146  Python-3.10.16 torch-2.7.0+cpu CPU (AMD Ryzen 5 5625U with Radeon Graphics)
YOLO11s-seg summary (fused): 113 layers, 10,097,776 parameters, 0 gradients, 35.5 GFLOPs
val: Fast image access  (ping: 0.00.0 ms, read: 1552.3419.9 MB/s, size: 325.0 KB)


val: Scanning C:\Users\tiess\PycharmProjects\Tygron\ModelResearch\Yolo\OnderzoekDataSubsetYolo\Val\labels.cache... 150 images, 9 backgrounds, 0 corrupt: 100%|██████████| 150/150 [00:00<?, ?it/s]
C:\Users\tiess\anaconda3\envs\machlearn\lib\site-packages\torch\utils\data\dataloader.py:665: UserWarning: 'pin_memory' argument is set as true but no accelerator is found, then device pinned memory won't be used.
  warnings.warn(warn_msg)
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):   0%|          | 0/10 [00:00<?, ?it/s]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...


                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  10%|█         | 1/10 [00:04<00:42,  4.74s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validati

                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  20%|██        | 2/10 [00:10<00:42,  5.34s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validati

                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95): 100%|██████████| 10/10 [00:48<00:00,  4.82s/it]


                   all        150       1067     0.0121     0.0637    0.00683    0.00203    0.00853      0.045    0.00459    0.00131
                person        141       1067     0.0121     0.0637    0.00683    0.00203    0.00853      0.045    0.00459    0.00131
Speed: 2.0ms preprocess, 265.3ms inference, 0.0ms loss, 4.3ms postprocess per image
Results saved to runs\segment\val36
mAP@0.5 (Seg) for yolo11s-seg.pt: 0.0046

Evaluating yolo11m-seg.pt
Ultralytics 8.3.146  Python-3.10.16 torch-2.7.0+cpu CPU (AMD Ryzen 5 5625U with Radeon Graphics)
YOLO11m-seg summary (fused): 138 layers, 22,396,992 parameters, 0 gradients, 123.3 GFLOPs
val: Fast image access  (ping: 0.10.0 ms, read: 1850.7104.2 MB/s, size: 324.0 KB)


val: Scanning C:\Users\tiess\PycharmProjects\Tygron\ModelResearch\Yolo\OnderzoekDataSubsetYolo\Val\labels.cache... 150 images, 9 backgrounds, 0 corrupt: 100%|██████████| 150/150 [00:00<?, ?it/s]
C:\Users\tiess\anaconda3\envs\machlearn\lib\site-packages\torch\utils\data\dataloader.py:665: UserWarning: 'pin_memory' argument is set as true but no accelerator is found, then device pinned memory won't be used.
  warnings.warn(warn_msg)
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):   0%|          | 0/10 [00:00<?, ?it/s]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...


                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  10%|█         | 1/10 [00:11<01:42, 11.44s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validati

                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  20%|██        | 2/10 [00:24<01:37, 12.14s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validati

                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95): 100%|██████████| 10/10 [01:53<00:00, 11.33s/it]


                   all        150       1067     0.0119     0.0581    0.00677    0.00206    0.00808     0.0394    0.00443    0.00117
                person        141       1067     0.0119     0.0581    0.00677    0.00206    0.00808     0.0394    0.00443    0.00117
Speed: 2.0ms preprocess, 701.0ms inference, 0.0ms loss, 2.9ms postprocess per image
Results saved to runs\segment\val37
mAP@0.5 (Seg) for yolo11m-seg.pt: 0.0044

Evaluating yolo11l-seg.pt
Ultralytics 8.3.146  Python-3.10.16 torch-2.7.0+cpu CPU (AMD Ryzen 5 5625U with Radeon Graphics)
YOLO11l-seg summary (fused): 203 layers, 27,646,272 parameters, 0 gradients, 142.2 GFLOPs
val: Fast image access  (ping: 0.00.0 ms, read: 1792.2493.8 MB/s, size: 303.0 KB)


val: Scanning C:\Users\tiess\PycharmProjects\Tygron\ModelResearch\Yolo\OnderzoekDataSubsetYolo\Val\labels.cache... 150 images, 9 backgrounds, 0 corrupt: 100%|██████████| 150/150 [00:00<?, ?it/s]
C:\Users\tiess\anaconda3\envs\machlearn\lib\site-packages\torch\utils\data\dataloader.py:665: UserWarning: 'pin_memory' argument is set as true but no accelerator is found, then device pinned memory won't be used.
  warnings.warn(warn_msg)
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):   0%|          | 0/10 [00:00<?, ?it/s]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...


                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  10%|█         | 1/10 [00:13<02:04, 13.79s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...


                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  20%|██        | 2/10 [00:28<01:54, 14.27s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...


                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95): 100%|██████████| 10/10 [02:11<00:00, 13.13s/it]


                   all        150       1067     0.0122      0.059    0.00658    0.00204     0.0108     0.0525    0.00591    0.00171
                person        141       1067     0.0122      0.059    0.00658    0.00204     0.0108     0.0525    0.00591    0.00171
Speed: 2.2ms preprocess, 832.6ms inference, 0.0ms loss, 1.4ms postprocess per image
Results saved to runs\segment\val38
mAP@0.5 (Seg) for yolo11l-seg.pt: 0.0059

Evaluating yolo11x-seg.pt
Ultralytics 8.3.146  Python-3.10.16 torch-2.7.0+cpu CPU (AMD Ryzen 5 5625U with Radeon Graphics)
YOLO11x-seg summary (fused): 203 layers, 62,094,528 parameters, 0 gradients, 319.0 GFLOPs
val: Fast image access  (ping: 0.00.0 ms, read: 1748.9502.0 MB/s, size: 320.1 KB)


val: Scanning C:\Users\tiess\PycharmProjects\Tygron\ModelResearch\Yolo\OnderzoekDataSubsetYolo\Val\labels.cache... 150 images, 9 backgrounds, 0 corrupt: 100%|██████████| 150/150 [00:00<?, ?it/s]
C:\Users\tiess\anaconda3\envs\machlearn\lib\site-packages\torch\utils\data\dataloader.py:665: UserWarning: 'pin_memory' argument is set as true but no accelerator is found, then device pinned memory won't be used.
  warnings.warn(warn_msg)
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):   0%|          | 0/10 [00:00<?, ?it/s]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validati

                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  10%|█         | 1/10 [00:25<03:46, 25.19s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validati

                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  20%|██        | 2/10 [00:50<03:24, 25.55s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validati

                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95): 100%|██████████| 10/10 [03:56<00:00, 23.70s/it]


                   all        150       1067     0.0109     0.0534    0.00587    0.00192    0.00845     0.0412    0.00453    0.00111
                person        141       1067     0.0109     0.0534    0.00587    0.00192    0.00845     0.0412    0.00453    0.00111
Speed: 2.4ms preprocess, 1523.5ms inference, 0.0ms loss, 2.5ms postprocess per image
Results saved to runs\segment\val39
mAP@0.5 (Seg) for yolo11x-seg.pt: 0.0045

Evaluating yolov9e-seg.pt
Ultralytics 8.3.146  Python-3.10.16 torch-2.7.0+cpu CPU (AMD Ryzen 5 5625U with Radeon Graphics)
YOLOv9e-seg summary (fused): 292 layers, 59,743,360 parameters, 0 gradients, 244.8 GFLOPs
val: Fast image access  (ping: 0.00.0 ms, read: 2066.1255.9 MB/s, size: 336.0 KB)


val: Scanning C:\Users\tiess\PycharmProjects\Tygron\ModelResearch\Yolo\OnderzoekDataSubsetYolo\Val\labels.cache... 150 images, 9 backgrounds, 0 corrupt: 100%|██████████| 150/150 [00:00<?, ?it/s]
C:\Users\tiess\anaconda3\envs\machlearn\lib\site-packages\torch\utils\data\dataloader.py:665: UserWarning: 'pin_memory' argument is set as true but no accelerator is found, then device pinned memory won't be used.
  warnings.warn(warn_msg)
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):   0%|          | 0/10 [00:00<?, ?it/s]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...


                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  10%|█         | 1/10 [00:35<05:17, 35.26s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...


                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95):  20%|██        | 2/10 [01:04<04:14, 31.80s/it]

WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...
WARNING Limiting validation plots to first 50 items per image for speed...


                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mAP50-95): 100%|██████████| 10/10 [04:32<00:00, 27.22s/it]


                   all        150       1067    0.00821     0.0206    0.00431     0.0012    0.00709     0.0178    0.00374      0.001
                person        141       1067    0.00821     0.0206    0.00431     0.0012    0.00709     0.0178    0.00374      0.001
Speed: 2.3ms preprocess, 1760.1ms inference, 0.0ms loss, 1.6ms postprocess per image
Results saved to runs\segment\val40
mAP@0.5 (Seg) for yolov9e-seg.pt: 0.0037


# MaP@0.5 (Mean average Precision met 0.5 confidence score) Resultaten
yolo11n-seg: 0.0074  
yolo11s-seg: 0.0046  
yolo11m-seg: 0.0044  
yolo11l-seg: 0.0059  
yolo11x-seg: 0.0045  
yolov9e-seg: 0.0037  

Opvallend is dat de prestaties van de modellen niet consistent verbeteren naarmate de modelgrootte toeneemt, wat doorgaans wel verwacht zou worden. Zo behaalt het kleinste model (yolo11n-seg) met 0.0074 de hoogste MaP@0.5-score, terwijl grotere modellen zoals yolo11x-seg en yolov9e-seg juist lagere scores laten zien.

Dit resultaat is opmerkelijk en suggereert dat schaalvergroting van het model in deze context niet automatisch leidt tot betere prestaties. Mogelijke verklaringen kunnen liggen in:
- Overfitting bij grotere modellen, vooral bij beperkte of suboptimale trainingsdata.
- Niet-geoptimaliseerde hyperparameters voor elk model.
- Mogelijke incompatibiliteit tussen de dataset en de capaciteiten van grotere modellen